src/Panel/Document.php

```diff
@@ -139,14 +127,16 @@ public static function assets(): array
     }
 
     /**
-     * Check for a custom css file from the
-     * config (panel.css)
+     * Check for a custom asset file from the
+     * config (e.g. panel.css or panel.js)
+     * @since 3.6.2
      *
+     * @param string $option asset option name
      * @return string|null
      */
-    public static function customCss(): ?string
+    public static function customAsset(string $option): ?string
     {
-        if ($css = kirby()->option('panel.css')) {
+        if ($css = kirby()->option($option)) {
             $asset = asset($css);
```

Comment:
It's not a big deal, but we'd better change the name of the `$css` variables. Because it can be confusing.